examples/TumblingUnconstrainedRod/Tumbling_Unconstrained_Rod_postprocessing.py

```diff
@@ -0,0 +1,309 @@
+import logging
+from elastica.utils import MaxDimension, Tolerance
+from elastica.external_forces import NoForces
+from elastica.typing import SystemType
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import cm
+from tqdm import tqdm
+from typing import Dict, Sequence
+
+
+class EndpointForces_with_time_factor(NoForces):
+
+    def __init__(self, start_force, end_force, time_factor):
+
+        super(EndpointForces_with_time_factor, self).__init__()
+        self.start_force = start_force
+        self.end_force = end_force
+        self.time_factor = time_factor
+
+    def apply_forces(self, system: SystemType, time=0.0):
+
+        factor = self.time_factor(time)
+
+        system.external_forces[..., 0] += self.start_force * factor
+        system.external_forces[..., -1] += self.end_force * factor
+
+
+class EndPointTorque(NoForces):
+    def __init__(self, torque, direction=np.array([0.0, 0.0, 0.0])):
+        super(EndPointTorque, self).__init__()
+        self.torque = torque * direction
+
+    def apply_torques(self, system: SystemType, time: np.float64 = 0.0):
+        n_elems = system.n_elems
+        if time < 1:
+            factor = 1
+        else:
+            factor = 0
+        system.external_torques[..., -1] += self.torque * factor
+
+
+class EndPointTorque_with_time_factor(NoForces):
```

Comment:
Change name to `EndPointTorquesWithTimeFactor`. Make sure these changes are reflected where they are used.